js/script.js

```diff
@@ -0,0 +1,1286 @@
+
+const str = '123456';
+const a = str.split('').reverse().join('');
+
+console.log(a);
+
+
+//Дана строка. Сделайте заглавным первый символ этой строки не используя цикл. Найдите два решения
+
+const str = 'hello';
+const a = str[0].toUpperCase() + str.substring(1);
+
+console.log(a);
+
+
+//Проверьте, что строка начинается на http://.
+
+const str = 'http://do';
+
+const result = str.startsWith('http://');
+
+console.log(result);
+
+
+//Проверьте, что строка заканчивается на .html
+
+const str = 'do.html';
+
+const result = str.endsWith('.html');
+
+console.log(result);
+
+
+//Дан массив с числами. Проверьте, что в этом массиве есть число 5. Если есть - выведите 'да', а если нет - выведите 'нет'.
+
+const a = [1, 2, 3, 4, 5];
+let result = '';
+
+for (let i = 0; i < a.length; i++) {
+  if (a[i] === 5){
+    result = 'yes';
+  } else {
+    result = 'no';
+  }
+}
+
+console.log(result);
+
+
+//Дано число, например 31. Проверьте, что это число не делится ни на одно другое число кроме себя самого и единицы. То есть в нашем случае нужно проверить, что число 31 не делится на все числа от 2 до 30. Если число не делится - выведите 'false', а если делится - выведите 'true'.
+
+function checkNum (a) {
+  if ((a % 1 === a) && (a % 31 === 1)) {
+    return false;
+  } else {
+    return true;
+  }
+}
+
+console.log(checkNum(31));
+
+
+//Дан массив с числами. Проверьте, есть ли в нем два одинаковых числа подряд. Если есть - выведите 'да', а если нет - выведите 'нет'
+
+const arr =[1, 1, 2];
+
+function checkNum () {
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[i] === arr[i + 1])
+    return (console.log('yes'));
+    }
+    return (console.log('no'));
+}
+
+checkNum (arr);
+
+
+//Заполните массив следующим образом: в первый элемент запишите '1', во второй '22', в третий '333' и так далее (два цикла) 
+
+let result = '';
+function getArr () {
+  for (let i = 1; i <= 5; i++) {
+    for (let j = 0; j < i; j++) {
+      result += i
+    }
+  }
+  return [result];
+}
+console.log(getArr());
+
+
+//Fill. Напишите функцию, которая заполняет новый массив предоставленным значением. 
+
+function fill (arraySize, value) {
+  let arr = [];
+
+  for(let i = 0; i < arraySize; i++) {
+    arr.push(value);
+  }
+
+  return arr;
+}
+
+const data = 3;
+const valueToFill = 'a';
+console.log(fill(data, valueToFill)); // ['a', 'a', 'a']
+
+
+//Reverse. Напишите функцию, которая разворачивает массив в обратном порядке. Пожалуйста, не используйте array.reverse(), чтобы сделать задачу более интересной.
+
+function reverse (array) {
+  const newArr = [];
+  
+  for (let i = array.length - 1; i >= 0; i = i - 1) {
+    newArr.push(array[i]);
+  }
+
+  return newArr;
+}
+
+const data = [1, 2, 3];
+console.log(reverse(data)); // [3, 2, 1]
+
+
+//Compact. Напишите функцию, которая очищает массив от нежелательных значений, таких как false, undefined, пустые строки, ноль, null.
+
+const compact = (array) => 
```

Comment:
не нужны скобки если 1 параметр
```suggestion
const compact = array => 
```